### DISCUSSION TITLE
the guide too hard to know how to use all functions

### DISCUSSION THREAD

**Discussion by x4r3x** [antilandjs/antiland.js#14] [General]
the guide too hard to know, i wanna make an automate texting for get Karma but i cannot understand how to use those function. btw your project very well, respect to you

**Comment by Calculamatrise** [comment]
Please refer to the [Wiki](https://github.com/Calculamatrise/antiland/wiki) that has been written as documentation for this library. It is a little outdated, but most of the methods are still the same. As long as you have basic knowledge in JavaScript, you should be able to understand it.
